### PR TITLE
Interbranch workflow

### PR DESCRIPTION
**Pull request by singhashish-wpf**
Sync changes between branches.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/wpf/pull/9821)

### CODE REVIEW

**Review by Kuldeep-MS — APPROVED** [review]

**Comment by pchaurasia14** [comment]
cc : @carlossanlop

File: github-merge-flow.jsonc.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        // Automate merging runtime release/6.0 branch into release/6.0-staging
+        "release/6.0":{
+            "MergeToBranch": "release/6.0-staging",
+            "ExtraSwitches": "-QuietComments"
+        },
+        // Automate merging runtime release/8.0 branch into release/8.0-staging
+        "release/8.0":{
+            "MergeToBranch": "release/8.0-staging",
+            "ExtraSwitches": "-QuietComments"
+        },
```
Comment (carlossanlop) [inline github-merge-flow.jsonc.txt:13]: This repo does not use staging branching. Are you planning on implementing it? If not, then remove the first two entries.

File: github-merge-flow.jsonc.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        // Automate merging runtime release/8.0-rc branches back to release/8.0
+        "release/8.0-rc":{
```
Comment (carlossanlop) [inline github-merge-flow.jsonc.txt:15]: 8.0 is already released. There's no need for 8.0-rc* branches. Remove this entry.

We had it in runtime a year ago, but never removed it.

File: github-merge-flow.jsonc.txt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            "MergeToBranch": "release/8.0",
+            "ExtraSwitches": "-QuietComments"
+        },
+        // Automate merging runtime release/9.0-rc1 branch back to release/9.0
+        "release/9.0-rc1":{
```
Comment (carlossanlop) [inline github-merge-flow.jsonc.txt:20]: RC1 is already out. Remove this entry.

**Pull request merged**